packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageMarketCard.tsx

```diff
@@ -4,26 +4,33 @@ import { THooks } from '../../../../types';
 import { DynamicLeverageTableColumnHeader } from './DynamicLeverageTableColumnHeader';
 import './DynamicLeverageMarketCard.scss';
 
-type TDynamicLeverageMarketCardProps = THooks.DynamicLeverage;
+type TDynamicLeverageMarketCardProps = {
+    data: THooks.DynamicLeverage[keyof THooks.DynamicLeverage]['volume']['data'];
+    displayName: string;
+    instruments: string[];
+    max: number;
+    min: number;
+};
 
 export const DynamicLeverageMarketCard: FC<TDynamicLeverageMarketCardProps> = ({
     data,
-    description,
-    leverage,
-    title,
+    displayName,
+    instruments,
+    max,
+    min,
 }) => (
     <div className='wallets-dynamic-leverage-modal__market'>
         <div className='wallets-dynamic-leverage-modal__market-title'>
             <WalletText align='center' data-testid='market_title' size='sm' weight='bold'>
-                {title}
+                {displayName}
             </WalletText>
-            {description && (
+            {!!instruments.length && (
                 <WalletText align='center' data-testid='description_title' fontStyle='italic' size='2xs'>
-                    {description}
+                    {`(${instruments.join(', ')})`}
```

Comment:
```suggestion
                    {instruments.join(', ')}
```

Reply:
Wanted to add round brackets around the instruments.

Reply:
👍🏼